runtime/compiler/rpc/J9Client.hpp

```diff
@@ -138,17 +138,39 @@ class J9ClientStream : J9Stream
       return _incompatibilityCount < INCOMPATIBILITY_COUNT_LIMIT;
       }
 
+   static void setServerUnavailable(OMRPortLibrary *portLibrary)
+      {
+      OMRPORT_ACCESS_FROM_OMRPORT(portLibrary);
+      if (_serverAvailable)
+         {
+         _serverAvailable = false;
+         _unavailableStartTime = omrtime_current_time_millis();
```

Comment:
Let's set `_unavailableStartTime` every time we call this function. Basically, we should be looking at the most recent event when we found the server unavailable and start the clock from that moment.